### PR TITLE
Another grab bag

### LINKED ISSUE #14: Figure out whether we want to manage addons via image streams
https://github.com/jim-minter/azure-helm/tree/master/pkg/addons/data/ImageStream.image.openshift.io

I guess we need to agree on how addon updates are going to be triggered.

### PR DESCRIPTION
**Pull request by jim-minter**
fixes #14 (stops using imagestreams for infra components)
makes a configurable for every image and sets imagePullPolicy: Always across the board
starts a stub for setting config based on the requested openshift version
starts work on waiting for deployments after helm upgrade (but may be buggy)

### CODE REVIEW

**Comment by jim-minter** [comment]
@pweil- @mjudeikis @kargakis ptal

**Comment by kargakis** [comment]
/lgtm

Follow-up issue to invalidate patch versions in Manifest or it's still
undecided?

**Comment by jim-minter** [comment]
> Follow-up issue to invalidate patch versions in Manifest or it's still undecided?

Will be in #60.

File: pkg/config/generate.go [hunk 1]
```diff
@@ -18,29 +18,64 @@ import (
...
 
-	c.MasterEtcdImage = "quay.io/coreos/etcd:v3.2.15"
-	c.MasterAPIImage = "docker.io/openshift/origin-control-plane:v3.10"
-	c.MasterControllersImage = "docker.io/openshift/origin-control-plane:v3.10"
-	c.BootstrapAutoapproverImage = "docker.io/openshift/origin-node:v3.10.0"
-	c.ServiceCatalogImage = "quay.io/kargakis/servicecatalog:kubeconfig" // TODO: "docker.io/openshift/origin-service-catalog:v3.10.0"
-	c.TunnelImage = "docker.io/jimminter/tunnel:latest"
-	c.SyncImage = "docker.io/jimminter/sync:latest"
-	c.TemplateServiceBrokerImage = "docker.io/openshift/origin-template-service-broker:v3.10"
+func selectContainerImagesOrigin(m *api.Manifest, c *Config) {
```
Comment (kargakis) [inline pkg/config/generate.go]: Nice, I also wanted to do this refactor.

**Pull request merged**